packs/data/spell-effects.db/spell-effect-wish-twisted-form-critical-failure.json

```diff
@@ -0,0 +1,99 @@
+{
+    "_id": "f5EOSVWSkG8RmXee",
+    "img": "systems/pf2e/icons/spells/wish-twisted-form.webp",
+    "name": "Spell Effect: Wish-Twisted Form (Critical Failure)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Wish-Twisted Form]{Wish-Twisted Form}</em>.</p>\n<p>Implemented Effects:</p>\n<ul>\n<li>The target gains weakness 5 to acid damage.</li>\n<li>The target takes a -10-foot status penalty to all its Speeds.</li>\n<li>The target takes a -1 status penalty to its saving throws.</li>\n<li>The target takes a -1 status penalty to AC.</li>\n</ul>\n<p>Unimplemented Effects:</p>\n<ul>\n<li>The target's resistances are reduced by 5.</li>\n</ul>\n<hr />\n<p><strong>Heightened (+1)</strong> Increase the weakness and the reduction to resistances by 1.</p>"
```

Comment:
The reference to acid damage should be changed, as the damage type is now a ChoiceSet.

Reply:
Still has a reference to acid damage.